src/analytics/Events.tsx

```diff
@@ -199,6 +202,10 @@ export enum PhoneVerificationEvents {
   phone_verification_input_help_continue = 'phone_verification_input_help_continue', // when the user presses continue on the help dialog to continue verification
 
   phone_verification_resend_message = 'phone_verification_resend_message', // when the user triggers a resend of the sms code
+
+  phone_verification_revoke_start = 'phone_verification_revoke_start', // when the revoke endpoint is called
+  phone_verification_revoke_success = 'phone_verification_revoke_success', // when the revoke endpoint returns successfully
+  phone_verification_revoke_error = 'phone_verification_revoke_error', // when the revoke endpoint returns an error
```

Comment:
i may have overdone it with the events. the `phone_verification` ones are related to calling the cloud function (which we can also get from the gcp logs), the `settings` ones are for button press. idk, happy to remove if anyone feels strongly but i thought it'd be easier to analyse the data if they're in the same place

Reply:
Seems fine 👍